packages/x-charts/src/internals/warning.ts

```diff
@@ -0,0 +1,17 @@
+export function buildWarning(
```

Comment:
I would probably just rename it to `buildOneTimeWarning` or `buildWarningOnce`

Reply:
It's a common piece of code we duplicate in each package.

Was introduced mostly to avoid code duplication in the data grid. We could do the renaming but in that case, would require to do it in all packages

Reply:
up to you, doesn't matter much 😅 

should we eventually have a shared X package? shouldn't be too hard to accomplish

Reply:
> should we eventually have a shared X package?

This has been discussed quite recently
I feel like we are only waiting for a use-case and someone to own the creation of the new package :laughing: 

@joserodolfofreitas @cherniavskii if you are OK, we can create it

Reply:
Agreed, we need a shared utils package 👍🏻